org.taskcoach.TaskCoach.yaml

```diff
@@ -0,0 +1,191 @@
+app-id: org.taskcoach.TaskCoach
+runtime: org.gnome.Platform
+runtime-version: '3.30'
+sdk: org.gnome.Sdk
+command: taskcoach
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --share=network
```

Comment:
What does it use network access for?

Reply:
From the FAQ I see two things, but I haven't used either myself.
- Attachments can be added to tasks, notes, and categories by dragging and dropping files, e-mail messages, or URL's onto a task, note or category.
- Tasks and notes can be synchronized via a Funambol server such as My Funambol.